### PR TITLE
Add zsh completion for 'service {create,update} --group-{add,rm}'

### PR DESCRIPTION
**Pull request by sdurrheimer**
ref. #25317

@albers Thx for the ping

Signed-off-by: Steve Durrheimer s.durrheimer@gmail.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
LGTM

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -1083,6 +1083,8 @@ __docker_service_subcommand() {
         "($help)*--constraint=[Placement constraints]:constraint: "
         "($help)--endpoint-mode=[Placement constraints]:mode:(dnsrr vip)"
         "($help)*"{-e=,--env=}"[Set environment variables]:env: "
+        "($help)*--group-add=[Add additional user groups to the container]:group:_groups"
+        "($help)*--group-rm=[Remove previously added user groups from the container]:group:_groups"
```
Comment (albers) [inline contrib/completion/zsh/_docker]: Is this correct? `--group-add` is available for both `docker service update` and `docker service create` but `--group-rm` is only available for `docker service update` (sorry for the misleading subject that I gave to my PR)

Comment (cpuguy83) [inline contrib/completion/zsh/_docker]: Ah yep, you are right.

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: Woops, my bad. Fixed !

**Comment by vdemeester** [comment]
LGTM 🐸
Thanks @sdurrheimer @albers

**Pull request merged**